drivers/net/ethernet/intel/idpf/idpf_eth_idc.h

```diff
@@ -0,0 +1,199 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/* Copyright (C) 2024 Intel Corporation */
+
+#ifndef _IDPF_ETH_IDC_H_
+#define _IDPF_ETH_IDC_H_
+
+#include <linux/auxiliary_bus.h>
+#include "idpf_eth_common.h"
+
+struct idpf_adapter;
+struct idpf_vc_xn_params;
+
+#define IDPF_MAX_Q				16
```

Comment:
These defines are not standard. They are pci driver specific resource management. So should be moved out of this header. They can be moved to eth_common.h i assume.

Reply:
Done!